### ISSUE TITLE
Wind rose malfunctioning

### ISSUE THREAD

**Issue by Osolemio44** [plotly/plotly.js#5793]
Not that plotly has a bug. Is that the example that you have set as a wind rose does not work well. That is to say, it is not a wind rose. Below I put a comparison of how a compass rose should work, and how the one you have put in the documentation / example works.
![plotly error](https://user-images.githubusercontent.com/55595063/124096354-c9a6dd80-da5a-11eb-9c38-6929cf4574f9.png)
In the following video I show you the difference with a wind rose that works well. Please note that the data are independent of each other and are represented on top of each other. While in the compass rose of plotly, they are totaled. Also, in a normal rose, when you turn off a lower data span, the scale changes. However, when doing it in plotly, the scale does not change because the data is wrong and the functioning is wrong. 

https://user-images.githubusercontent.com/55595063/124102309-86e80400-da60-11eb-8bd2-d76998bc55ea.mp4

There you have the live examples to try it yourself:

example 1 (plotly): [https://plotly.com/javascript/wind-rose-charts/](https://plotly.com/javascript/wind-rose-charts/)
example 2 (other chart): [https://jscharting.com/examples/chart-types/radar-polar/wind-rose/](https://jscharting.com/examples/chart-types/radar-polar/wind-rose/)
example 3 ( other chart): [https://www.makeapie.com/editor.html?c=xa-f-aLlNA](https://www.makeapie.com/editor.html?c=xa-f-aLlNA)
I do not doubt that this chart is very useful for some people, but of course it is not a wind rose and therefore it is totally unusable for that purpose.
Can someone tell me if a wind rose can be made that works correctly using plotly?
If yes, could someone change the example in the documentation to see how?
I'm totally new to using plotly and have no idea how to make a wind rose.

**Comment by nicolaskruchten** [comment]
You're right, in our Javascript documentation this is incorrect. It's correct in the Python version of the docs, however: https://plotly.com/python/wind-rose-charts/

The key is the `barmode: "overlay"` attribute in the JS example (and the data prep). We'll try to rework it to function like the Python version, but in the meantime, dropping that attribute will allow your bars to stack as they do in the Python version :)

**Comment by Osolemio44** [comment]
Hey!. Thanks for your quick answer.
I just removed the barmode: "overlay" attribute. It seems that the operation is now correct. However, it represents the data in an inverted way. In other words, the 11-14ms fringes are represented where the <5ms should be and vice versa.
